format/FlightSql.proto

```diff
@@ -1262,6 +1272,16 @@ message ActionClosePreparedStatementRequest {
  * Represents a SQL query. Used in the command member of FlightDescriptor
  * for the following RPC calls:
```

Comment:
Why does this have the same description as CommandGetTables? Do we have two different messages for doing the same thing?

Reply:
The message itself is different, what is similar here is the response since the column metadata can be retrieved via both command